regression/README.md

```diff
@@ -34,106 +22,9 @@ you will learn how to use MLflow Recipes to
+All of these can be done with Jupyter notebook or on the Databricks environment.
 Finally, challenge yourself to build a better model. Try the following:
 - Find a better data source with more training data and more raw feature columns.
 - Clean the dataset to make it less noisy.
 - Find better feature transformations.
 - Fine tune the hyperparameters of the model.
-
-## Installation instructions
-Follow the [MLflow Recipes installation guide](https://mlflow.org/docs/latest/recipes.html#installation).
-You may need to install additional libraries for extra features:
-- [Hyperopt](https://pypi.org/project/hyperopt/)  is required for hyperparameter tuning.
-- [PySpark](https://pypi.org/project/pyspark/)  is required for distributed training or to ingest Spark tables.
-- [Delta](https://pypi.org/project/delta-spark/) is required to ingest Delta tables.
-These libraries are available natively in the [Databricks Runtime for Machine Learning](https://docs.databricks.com/runtime/mlruntime.html).
-
-## Log to the designated MLflow Experiment
-To log recipe runs to a particular MLflow experiment:
-1. Open `profiles/databricks.yaml` or `profiles/local.yaml`, depending on your environment.
-2. Edit (and uncomment, if necessary) the `experiment` section, specifying the name of the
-   desired experiment for logging.
-
-## Development Environment -- Databricks
-[Sync](https://docs.databricks.com/repos.html) this repository with
-[Databricks Repos](https://docs.databricks.com/repos.html) and run the `notebooks/databricks`
-notebook on a Databricks Cluster running version 11.0 or greater of the
-[Databricks Runtime](https://docs.databricks.com/runtime/dbr.html) or the
-[Databricks Runtime for Machine Learning](https://docs.databricks.com/runtime/mlruntime.html)
-with [workspace files support enabled](https://docs.databricks.com/repos.html#work-with-non-notebook-files-in-a-databricks-repo).
-
-**Note**: When making changes to recipes on Databricks,
-it is recommended that you edit files on your local machine and
-use [dbx](https://docs.databricks.com/dev-tools/dbx.html) to sync them to Databricks Repos, as
-demonstrated [here](https://mlflow.org/docs/latest/recipes.html#usage)
-
-**Note**: data profiles display in step cards are not visually compatible with dark theme.
-Please avoid using the dark theme if possible.
-
-### Accessing MLflow recipe Runs
-You can find MLflow Experiments and MLflow Runs created by the recipe on the
-[Databricks ML Experiments page](https://docs.databricks.com/applications/machine-learning/experiments-page.html#experiments).
-
-## Development Environment -- Local machine
-### Jupyter
-
-1. Launch the Jupyter Notebook environment via the `jupyter notebook` command.
-2. Open and run the `notebooks/jupyter.ipynb` notebook in the Jupyter environment.
-
-**Note**: data profiles display in step cards are not visually compatible with dark theme.
-Please avoid using the dark theme if possible.
-
-### Command-Line Interface (CLI)
```

Comment:
Is there a reason we are removing this? I think users could checkout this example repo and try out the CLI commands as well. So having this user could be helpful to the users. What do you think?

Reply:
Never mind, I saw that it was added here: https://github.com/mlflow/recipes-examples/pull/10/files